libs/datamodel/connectors/dml/src/default_value.rs

```diff
@@ -23,6 +23,11 @@ impl DefaultValue {
         }
     }
 
+    /// Does this match @default(autoincrement())?
```

Comment:
the test below answers that right?

Reply:
Ah, it was meant as way to document the function :p the returned boolean is the answer. I should maybe formulate that differently.

Reply:
Ah ok